src/pages/Login/index.js

```diff
@@ -0,0 +1,30 @@
+import "./styles.less";
+import template from "./template.hbs";
+import {AbstractComponent} from "../../components/AbstractComponent";
+import {Form} from "../../components/Form";
+
+export class LoginPage extends AbstractComponent {
+
+    get form() {
+        return new Form({
+            elements: [
+                {
+                    label: "Логин",
+                    name: "login"
+                },
+                {
+                    label: "Пароль",
+                    name: "password"
+                },
```

Comment:
```suggestion
            elements: [
                {
                    label: "Логин",
                    name: "login",
                },
                {
                    label: "Пароль",
                    name: "password",
                },
```
В таких местах лучше ставить `,`. Тут можно почитать почему
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas